### PR TITLE
Add better support for Eaton UPS

### PR DESCRIPTION
**Pull request by mathieu-artic**
image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: tests/data/eatonups_eaton-connectups.json [hunk 1]
```diff
@@ -0,0 +1,1221 @@
...
+                    "sysName": null,
+                    "sysObjectID": ".1.3.6.1.4.1.534.1",
+                    "sysDescr": "ConnectUPS Web/SNMP Card V4.36",
+                    "sysContact": "<private>",
+                    "version": "FP: 0.13  INV: 1.29",
+                    "hardware": "PW9130 1000VA-R",
+                    "features": null,
+                    "os": "eatonups",
+                    "type": "power",
+                    "serial": null,
```
Comment (Jellyfrog) [inline tests/data/eatonups_eaton-connectups.json:15]: No serial to be found while at it?

Comment (mathieu-artic) [reply tests/data/eatonups_eaton-connectups.json]: An update with the serial number, not perfect but it is present ...
image

File: includes/definitions/discovery/eatonups.yaml [hunk 2]
```diff
@@ -1,8 +1,9 @@
-mib: XUPS-MIB:EATON-OIDS:EATON-EMP-MIB
+mib: XUPS-MIB:EATON-OIDS:EATON-EMP-MIB:UPS-MIB
 modules:
     os:
         hardware: XUPS-MIB::xupsIdentModel.0
         version: XUPS-MIB::xupsIdentSoftwareVersion.0
+        serial: UPS-MIB::upsIdentName.0
```
Comment (Jellyfrog) [inline includes/definitions/discovery/eatonups.yaml:6]: Maybe this to cut it a bit;
`serial_regex: '/ID: (?<serial>[^ ]+)/'`

Comment (mathieu-artic) [reply includes/definitions/discovery/eatonups.yaml:6]: Works perfectly ! thx !! 
image

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**